pkgs/development/embedded/teensy-loader-cli/default.nix

```diff
@@ -6,15 +6,15 @@
 , libusb-compat-0_1
 }:
 
-stdenv.mkDerivation rec {
+ stdenv.mkDerivation (finalAttrs: {
```

Comment:
```suggestion
stdenv.mkDerivation {
```
we don't need to use finalAttrs here

Reply:
Then how should I reuse the version attribute for `src.rev`?